docs/trex_security.md

```diff
@@ -127,7 +127,7 @@ Or use site-relative or protocol-agnostic links (that is, where the protocol is
 ```html
 
     <!-- Extensions Library ) -->
-    <script src="../../lib/tableau-extensions-0.8.0.js"></script>
+    <script src="../../lib/tableau.extensions.1.2.0.js"></script>
```

Comment:
should this be a reference to latest?

Reply:
@d45

Reply:
@seanmakesgames  Good catch. That makes more sense.  Thanks!

Reply:
I checked in a fix to change this to `1.latest.js`